CONTRIBUTING.adoc

```diff
@@ -246,6 +246,27 @@ Be aware that some of this content might already be present in the Jenkins User
 
 The documentation pages can use the same metadata (`title`, `description`, `opengraph:image`) as blog posts.
 
+* When creating or adding documentation, make sure that every numbered line contains a complete sentence, including an ending punctuation mark (`.`, `?`, `!`).
+Formatting your documentation this way makes it easier for reviewers to read over and provide feedback. 
++
+----
+The full sentence structure in a line makes it easier to read.
+Otherwise, sentences are broken up over multiple lines or multiple sentences are combined into one line.
+In cases of code examples or output, this is not necessary and should be formatted normally.
+----
+
+* When creating documentation, avoid future tense as much as possible.
+** Users should be able to approach documentation as though they are performing the actions as they read.
+This makes the text more active and encourages more interaction with the documentation itself.
+
+* Be sure to provide sample output when providing commands or code.
+This helps ensure readers can assess their work as they progress through the documentation.
+
+* Always use the correct naming for Jenkins temrinology.
+We have been working on updating the terminology throughout Jenkins documentation to align with the link:/sigs/advocacy-and-outreach/#inclusive-naming[inclusive naming initiative].
+
+** If you find instances of existing documentation where this should be updated, submit a PR with the fix, and we can make sure that the documentation is corrected.
```

Comment:
Perhaps useful to mention necessary exceptions (e.g. internal class names retained for compatibility, UI labels that haven't been updated in the software, etc.), because we had braindead PRs "fixing" those occurrences before.

Reply:
Agreed, thanks for calling this out @daniel-beck. I'll be sure to include that here.